[2025-07-28T18:38:35.635+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-28T18:38:35.686+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: financial_etl_pipeline.load_data manual__2025-07-28T18:38:29.549898+00:00 [queued]>
[2025-07-28T18:38:35.699+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: financial_etl_pipeline.load_data manual__2025-07-28T18:38:29.549898+00:00 [queued]>
[2025-07-28T18:38:35.701+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2025-07-28T18:38:36.018+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): load_data> on 2025-07-28 18:38:29.549898+00:00
[2025-07-28T18:38:36.068+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'financial_etl_pipeline', 'load_data', 'manual__2025-07-28T18:38:29.549898+00:00', '--job-id', '149', '--raw', '--subdir', 'DAGS_FOLDER/financial_etl_dag.py', '--cfg-path', '/tmp/tmpo9ed_5pk']
[2025-07-28T18:38:36.081+0000] {standard_task_runner.py:91} INFO - Job 149: Subtask load_data
[2025-07-28T18:38:36.089+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=10348) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-28T18:38:36.091+0000] {standard_task_runner.py:63} INFO - Started process 10350 to run task
[2025-07-28T18:38:36.251+0000] {task_command.py:426} INFO - Running <TaskInstance: financial_etl_pipeline.load_data manual__2025-07-28T18:38:29.549898+00:00 [running]> on host c9a4a3294b33
[2025-07-28T18:38:36.373+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='financial_etl_pipeline' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-28T18:38:29.549898+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-28T18:38:29.549898+00:00'
[2025-07-28T18:38:36.375+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-28T18:38:36.531+0000] {logging_mixin.py:188} INFO - Data loaded into PostgreSQL 'transactions' table.
[2025-07-28T18:38:36.533+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-28T18:38:36.534+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-28T18:38:36.546+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=financial_etl_pipeline, task_id=load_data, execution_date=20250728T183829, start_date=20250728T183835, end_date=20250728T183836
[2025-07-28T18:38:36.594+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-28T18:38:36.617+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-28T18:38:36.621+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
